src/pages/ScholarshipForm.jsx

```diff
@@ -38,25 +38,31 @@ function ScholarshipForm() {
     );
   }
 
+  function requiredDeadlineField(id, label) {
```

Comment:
This works but I was thinking more of a separate file so the component could be reused in another form.